### PR TITLE
[Examples] WiFi and Ethernet examples organization

### PR DESCRIPTION
**Pull request by pedrominatel**
## Summary

This PR is to change the WiFi and Ethernet examples organization in a separated folder. Two folders were created for WiFi and Ethernet examples.

## Impact

WiFi examples now are in a different folder location. This may cause some impacts on external references to the examples.

### CODE REVIEW

**Comment by pedrominatel** [comment]
PTAL @me-no-dev and evaluate the risks and pros and cons of this change.

**Comment by me-no-dev** [comment]
There are no risks other than maybe breaking some links somewhere on the web.

File: libraries/WiFi/library.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name=WiFi
+name=WiFi and Ethernet
```
Comment (Rotzbua) [inline libraries/WiFi/library.properties:1]: This breaks dependencies, do not change the name.

**Review by Rotzbua — CHANGES_REQUESTED** [review]

**Comment by pedrominatel** [comment]
PTAL @me-no-dev

**Pull request merged**